configdefinitions/src/vespa/dispatch.def

```diff
@@ -23,6 +23,13 @@ distributionPolicy enum { ROUNDROBIN, ADAPTIVE } default=ROUNDROBIN
 ## don't use it if you don't (really) mean it.
 maxHitsPerNode int default=2147483647
 
+## Probability for getting the correct topK documents.
+## A value of 1.0 will ask all partitions for topK documents.
```

Comment:
Here perhaps we should say "for the number of documents specified by the hits parameter".
In general it's a bit strange that "hits" is suddenly called "topK" just because we optimize, but otoh it makes it easier to search for this :-)

Reply:
Improved